README.md

```diff
@@ -10,7 +10,7 @@ In the European Union, this extension's fields correspond to [eForms BG-611 (Con
 
 ## Codelists
 
-The `financeType.csv` codelist is based on the list on [Page 57 of the World Bank PPP Disclosure Framework](https://thedocs.worldbank.org/en/doc/143671469558797229-0100022016/original/FrameworkPPPDisclosure071416.pdf#page=57).
+The `financingPartyType.csv` codelist is based on the list on [Page 57 of the World Bank PPP Disclosure Framework](https://thedocs.worldbank.org/en/doc/143671469558797229-0100022016/original/FrameworkPPPDisclosure071416.pdf#page=57).
```

Comment:
We've diverged so much, let's just delete this section.

Reply:
Done in https://github.com/open-contracting-extensions/ocds_finance_extension/pull/26/commits/73c7856e8130bd1dcdb636823d5b55b2204b2e3c